### ISSUE TITLE
Transformers: Human Alliance shooting in letters not work.

### ISSUE THREAD

**Issue by dante3732** [argonlefou/DemulShooter#233]
Shooting letters at the end of the game does not work.
![TF_Gun_R_Ring_dumped_2024_07_20_18_24_57_619](https://github.com/user-attachments/assets/70fd62c1-3d09-44f1-b9ba-850331ede73a)

**Comment by argonlefou** [comment]
Can I get more details ?
Type of gun used (analog, sinden, gun4ir, wii mote) ?

**Comment by dante3732** [comment]
> Can I get more details ? Type of gun used (analog, sinden, gun4ir, wii mote) ?

Wii Remote. It is the trigger that does not work, while it worked the whole game without problems.

**Comment by argonlefou** [comment]
Whoops sorry I must have been half asleep when answering....I was thinking about the latest Transformers game (Shadow Rising 😅)

Ok thanks for the details, I'll try again with the good Transformer game...

**Comment by argonlefou** [comment]
> > Can I get more details ? Type of gun used (analog, sinden, gun4ir, wii mote) ?
> 
> Wii Remote. It is the trigger that does not work, while it worked the whole game without problems.

Whoops sorry I must have been half asleep when answering....I was thinking about the latest Transformers game (Shadow Rising 😅)

Ok thanks for the details, I'll try again with the good Transformer game...

**Comment by argonlefou** [comment]
> > Can I get more details ? Type of gun used (analog, sinden, gun4ir, wii mote) ?
> 
> Wii Remote. It is the trigger that does not work, while it worked the whole game without problems.

Whoops sorry I must have been half asleep when answering....I was thinking about the latest Transformers game (Shadow Rising 😅)

Ok thanks for the details, I'll try again with the good Transformer game...

Can you confirm you're using Jconfig + DemulShooter for inputs ?

**Comment by dante3732** [comment]
> Can you confirm you're using Jconfig + DemulShooter for inputs ?

Yep. I not using teknoparrot for this game, because it have crashes on my AMD.

**Comment by argonlefou** [comment]
Ok thanks

IF (it's a big one) I can find a way to skip the game directly to the name entry screen, I will be able to have a look

If not, it would be too long to play the game each time to test a few seconds during the screen visibility
